docs/source/ci-cd.mdx

```diff
@@ -173,7 +178,7 @@ echo "$HOME/.rover/bin" >> $GITHUB_PATH
 
 <Note>
 
-Because the `rover config auth` command is interactive, you need to [authenticate using an environment variable](./configuring#with-an-environment-variable) in your project settings.
+Because the `rover config auth` command is interactive, you need to [authenticate using an environment variable](./configuring#with-an-environment-variable) in your project settings. 
```

Comment:
```suggestion
Because the `rover config auth` command is interactive, you need to [authenticate using an environment variable](./configuring#with-an-environment-variable) in your project settings.
```